lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -50,9 +50,6 @@ def install!
                   create_info_plist_file(target.info_plist_path, native_target, target.version, target.platform)
```

Comment:
do you also want to remove the manual plist generation step?

Reply:
yes, this `unless` should be here - since the static frameworks are linked at build time, a plist is not needed.

Reply:
makes sense, it was there before static frameworks were a thing.